Makefile

```diff
@@ -1,9 +1,12 @@
 .PHONY: build push
 
 build: 
-	docker-compose build --parallel
-
-push:
-	docker-compose push
-
-
+	docker buildx bake \
```

Comment:
Could you add a `make` command for building https://github.com/okteto/devenv/blob/main/docker-compose.rust.yml and https://github.com/okteto/devenv/blob/main/docker-compose.dev.yml?
It's not automated on CI because these images takes very long and they usually timeout, but having a `make` command will be handly to build those image in ARM easily

Reply:
Added!